stock_manual_transfer/i18n/es.po

```diff
@@ -0,0 +1,420 @@
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__activity_type_id
+msgid "Next Activity Type"
+msgstr "Tipo de la siguiente actividad"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__message_needaction_counter
+msgid "Number of Actions"
+msgstr "Número de acciones"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__message_has_error_counter
+msgid "Number of errors"
+msgstr "Número de errores"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,help:stock_manual_transfer.field_stock_manual_transfer__message_needaction_counter
+msgid "Number of messages which requires an action"
+msgstr "Número de mensajes que requieren una acción"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,help:stock_manual_transfer.field_stock_manual_transfer__message_has_error_counter
+msgid "Number of messages with delivery error"
+msgstr "Número de mensajes con error de entrega"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,help:stock_manual_transfer.field_stock_manual_transfer__message_unread_counter
+msgid "Number of unread messages"
+msgstr "Número de mensajes sin leer"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__date_planned
+msgid "Planned Date"
+msgstr "Fecha planeada"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__route_id
+msgid "Preferred Route"
+msgstr "Ruta preferida"
+
+#. module: stock_manual_transfer
+#: model:ir.model,name:stock_manual_transfer.model_procurement_group
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__procurement_group_id
+msgid "Procurement Group"
+msgstr "Grupo de aprovisionamiento"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer_line__product_id
+msgid "Product"
+msgstr "Producto"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer_line__product_uom_category_id
+msgid "Product's unit of measure category"
+msgstr "Categoria de unidades de medida del producto"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer_line__product_uom_qty
+msgid "Quantity"
+msgstr "Cantidad"
+
+#. module: stock_manual_transfer
+#: model:ir.model.fields,field_description:stock_manual_transfer.field_stock_manual_transfer__activity_user_id
+msgid "Responsible User"
+msgstr "Usuario responsable"
+
+#. module: stock_manual_transfer
+#: model:stock.location.route,name:stock_manual_transfer.demo_wh_route_reception
+msgid "Route Reception"
+msgstr "Recepción de ruta"
```

Comment:
Ruta de recepción

But in English is wrong.

Just name it reception, because "route" is redundant, as they are all routes

Reply:
Not fully applied, not renamed in English